gcn/notices/burstcube/Alert.example.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/burstcube/alert.schema.json",
+  "alert_datetime": "2024-06-01T00:05:00.00Z",
+  "alert_tense": "current",
+  "alert_type": "initial",
+  "mission": "BurstCube",
+  "record_number": 1,
+  "trigger_time": "2024-06-01T00:00:00.0Z",
+  "event_name": ["GRB240601A"],
+  "id": ["b240601000"],
+  "ra": 232.0,
+  "dec": -53.1,
+  "uncertainty_shape": "circle",
+  "ra_uncertainty": [-4.6, 6.1],
+  "dec_uncertainty": [-3.4, 5.6],
+  "systematic_included": false,
+  "healpix_url": "https://heasarc.gsfc.nasa.gov/burstcube/trigger/b240601000/current/b240601000_locmap.fits",
+  "healpix_file": "stuff",
+  "rate_snr": 7.2,
+  "rate_duration": 1.024,
+  "rate_energy_range": [50, 300],
+  "classification": { "GRB": 0.95, "SF": 0.03, "Local Particles": 0.02 },
+  "t90": 2.3,
+  "t90_error": [0.1, 0.1],
```

Comment:
We should also describe the -+ sign properly, is it already in definition, or one should provide proper sign.